code_challenges/hacker_rank/finding_the_percentage.py

```diff
@@ -0,0 +1,15 @@
+def finding_the_percentage(name_and_grades, student_query):
+    average_grade = (sum(name_and_grades[student_query])) / len(
```

Comment:
`sum(name_and_grades[student_query])` only returns a single value so it doesn't benefit from wrapping in `( )`